.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,83 @@
+name: Feature request
+description: Suggest a new capability, API, or chain integration for xchainjs-lib.
+title: "[Feature] "
+labels: ["enhancement"]
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)